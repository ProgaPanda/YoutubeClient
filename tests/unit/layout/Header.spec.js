import { shallowMount } from '@vue/test-utils';
import HeaderLayout from '@/layout/Header.layout.vue';

describe('HeaderLayout', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(HeaderLayout, {
      stubs: ['router-link'],
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
