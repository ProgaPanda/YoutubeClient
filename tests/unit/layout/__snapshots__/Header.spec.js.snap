// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLayout renders correctly 1`] = `
<header
  class="header"
>
  <div
    class="header__content"
  >
    <router-link-stub
      class="header__logo"
      to="/"
    >
      YT
      <span>
        client
      </span>
    </router-link-stub>
     
    <searchbar-stub />
  </div>
</header>
`;
